db/migrate/20141210173220_create_docker_tables.rb

```diff
@@ -8,25 +8,15 @@ def up
     end
     create_table :katello_docker_tags do |t|
       t.string :name, :limit => 255
-      t.integer :docker_image_id
       t.integer :repository_id
       t.timestamps
     end
     add_index :katello_docker_images, :uuid, :unique => true
-    add_foreign_key :katello_docker_tags, :katello_docker_images, :column => "docker_image_id"
     add_foreign_key :katello_docker_tags, :katello_repositories, :column => "repository_id"
-
-    add_index :katello_docker_tags, [:docker_image_id, :repository_id, :name],
-              :name => :docker_tag_docker_image_repo_name, :unique => true
```

Comment:
Should his index be dropped?

Reply:
we probably still want this index, just not with docker_image_id (maybe just re-create it without tha tone column in a later migration)